modules/Forum/pages/forum/edit.php

```diff
@@ -77,6 +76,25 @@
     Redirect::to(URL::build('/forum/topic/' . urlencode($post_id)));
 }
 
+/**
+ * @param array $labels  of label ids
+ * @param array $user_groups array of a user their group ids
+ * @return array An array of the ids of the labels the user has access to
+ */
+function getAccessibleLabels(array $labels, array $user_groups): array {
```

Comment:
This doesn't feel like the right location for a function but I didn't know where else to put it... So I just put it here 😅

Reply:
Probably the Forum class? modules/Forum/classes